tests/tests_integration/test_api/test_datapoint_subscriptions.py

```diff
@@ -261,10 +264,14 @@ def test_iterate_data_subscription_datapoints_added(
             assert batch.subscription_changes.added[0].external_id == time_series_external_ids[0]
 
             existing_data = cognite_client.time_series.data.retrieve_dataframe(external_id=time_series_external_ids[0])
+            if existing_data.empty:
+                start = pd.Timestamp("now")
+            else:
+                start = existing_data.index[-1] + pd.Timedelta("1d")
```

Comment:
This seems like bootstrapping the time series with at least one datapoint? I think that belongs in a fixture!

Reply:
This I am more skeptical about. If it is only used in one test, moving it to a fixture seems to obfuscate.

Reply:
The test is using a random xid (see fixture `time_series_external_ids`), meaning all these xids should have an existing datapoint. Anyway, approving 👍